buyer-api/src/blockchain/balance.js

```diff
@@ -74,14 +70,15 @@ export const hasEnoughBalance = async (address) => {
  */
 export const hasEnoughBatPayBalance = async (id, amount = minBatPay) => {
   const batPay = await getBatPayBalance(id);
-  const enough = batPay.gt(amount);
+  const enough = batPay.gt(toBN(amount));
 
   if (!enough) {
     logger.error(`
     BatPay.Account '${id}' does not have enough WIB.
     Current balance: ${batPay} WIB
     Minimum balance: ${amount} WIB
     `);
+    return false;
```

Comment:
I think you can remove this, enough would be false, and you are returning it just below 👇